docs/articles/internal/getting-started.md

```diff
@@ -1,6 +1,7 @@
 ---
 uid: articles-internal-getting-started
 title: Getting Started
+name: Getting Started
```

Comment:
out of curiosity: why do they need the name?

Reply:
toc autogenerator that i used required name for them (i was too lazy to write tocs manually)